src/content/en/2019/css.md

```diff
@@ -0,0 +1,251 @@
+---
+part_number: I
+chapter_number: 2
+title: CSS
+authors: [una, argyleink]
+reviewers: [meyerweb, huijing]
+---
+
+## Introduction
```

Comment:
This is looking great! The only other recommendation I have re content is to have a corresponding Conclusion section where you wrap it all up.